inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp

```diff
@@ -139,6 +139,8 @@ bool ngraph::pass::ConvertOpSet1ToLegacy::run_on_function(std::shared_ptr<ngraph
     anchor->add_matcher<ngraph::pass::ConvertLSTMSequenceMatcher>();
     anchor->set_name("ngraph::pass::LegacyConversions");
 
+    manager.register_pass<ngraph::pass::ConstantFolding>();
```

Comment:
CF is a very time consuming operation. If we know which pass requires CF let's do it manually. You can find an example here: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/transformations/src/transformations/common_optimizations/lin_op_sequence_fusion.cpp#L17

Reply:
Started to write the transformation to fold input "K" for `TopK`.